src/backend/pipeline/cont_plan.c

```diff
@@ -119,6 +119,45 @@ get_worker_plan(ContinuousView *view)
 	return get_plan_from_stmt(view->id, (Node *) selectstmt, view->query, false);
 }
 
+PlannedStmt*
+GetContinuousViewPlan(ContinuousView *view)
```

Comment:
We call what's being retrieved here the "overlay" view, so this should probably be named something like `GetContinuousViewOverlayPlan`.

Additionally, this is basically doing the same work that retrieving the combiner plan requires, which isn't necessary for getting the overlay plan. It's basically identical to getting the worker plan, but with passing a pointer to `TransformSelectStmtForContProcess`:

```
    TransformSelectStmtForContProcess(view->matrel, selectstmt, &overlay_stmt, Worker);
    return get_plan_from_stmt(view->id, (Node *) overlay_stmt, view->query, false);
```